services/appflowy-collaborate/src/metrics.rs

```diff
@@ -85,6 +91,27 @@ impl CollabRealtimeMetrics {
   }
 }
 
+pub(crate) fn spawn_metrics<S>(metrics: Arc<CollabRealtimeMetrics>, storage: Arc<S>)
```

Comment:
This was also removed in #770 but that part was not just copied from removed structure, it was periodically sampled and therefore should stay.